charts/prometheus-adapter/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus-adapter
-version: 4.6.0
+version: 4.6.1
```

Minor bump for new features
```suggestion
version: 4.7.0
```